src/_igraph/convert.c

```diff
@@ -2684,6 +2684,73 @@ int igraphmodule_PyObject_to_vector_int_ptr_t(PyObject* list, igraph_vector_ptr_
   return 0;
 }
 
+/**
+ * \ingroup python_interface_conversion
+ * \brief Converts a Python list of lists to an \c igraph_vector_list_t
+ *        (containing \c igraph_vector_int_t items).
+ *
+ * \param o the Python object representing the list of lists
+ * \param m the address of an uninitialized \c igraph_vector_list_t
+ * \return 0 if everything was OK, 1 otherwise. Sets appropriate exceptions.
+ */
+int igraphmodule_PyObject_to_vector_list_t(PyObject* list, igraph_vector_list_t* vec) {
+  PyObject *it, *item;
+  igraph_vector_t *subvec;
+
+  if (PyUnicode_Check(list)) {
+    PyErr_SetString(PyExc_TypeError, "expected iterable (but not string)");
+    return 1;
+  }
+
+  it = PyObject_GetIter(list);
+  if (!it) {
+    return 1;
+  }
+
+  if (igraph_vector_list_init(vec, 0)) {
+    igraphmodule_handle_igraph_error();
+    Py_DECREF(it);
+    return 1;
+  }
+
+  /* TODO: what about this? is memory managed already? */
+  //IGRAPH_VECTOR_PTR_SET_ITEM_DESTRUCTOR(vec, igraph_vector_int_destroy);
+  while ((item = PyIter_Next(it)) != 0) {
+    /* FIXME: is this even the right way to do this? */
+    subvec = igraph_Calloc(1, igraph_vector_t);
```

Comment:
igraph_Calloc is deprecated, use IGRAPH_CALLOC

Reply:
yep